redirects/linux.conf

```diff
@@ -0,0 +1,344 @@
+        # Top level redirects from NuDocs 1 to NuDocs 2
+        # =============================================
+
+        location /minio/baremetal/introduction/minio-overview.html {
+             return 301 /docs/linux/operations/concepts.html;
+        }
+
+        location /minio/baremetal/concepts/erasure-coding.html {
+             return 301 /docs/linux/operations/concepts/erasure-coding.html;
+        }
+
+        location /minio/baremetal/quickstart/linux.html {
+             return 301 /docs/linux/operations/index.html;
+        }
+
+
+        # Deploy/Manage Redirects from NuDocs 1 to NuDocs 2
+        # =================================================
+
+        location /minio/baremetal/installation/deployment-and-management.html {
+             return 301 /docs/linux/operations/installation.html;
+        }
+
+        location /minio/baremetal/installation/deploy-minio-distributed.html {
+             return 301 /docs/linux/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.html;
+        }
+
+        location /minio/baremetal/installation/deploy-minio-single-node-single-drive.html {
+             return 301 /docs/linux/operations/install-deploy-manage/deploy-minio-single-node-single-drive.html;
+        }
+
+        location /minio/baremetal/installation/expand-minio-distributed.html {
+             return 301 /docs/linux/operations/install-deploy-manage/expand-minio-deployment.html;
+        }
+
+        location /minio/baremetal/installation/upgrade-minio.html {
+             return 301 /docs/linux/operations/install-deploy-manage/upgrade-minio-deployment.html;
+        }
+
+        location /minio/baremetal/installation/restore-minio.html {
+             return 301 /docs/linux/operations/data-recovery.html;
+        }
+
+        location /minio/baremetal/installation/decommission-pool.html {
+             return 301 /docs/linux/operations/install-deploy-manage/decommission-server-pool.html;
+        }        
+
+
+        # Security Redirects from NuDocs 1 to NuDocs 2
+        # ============================================
+
+        location /minio/baremetal/security/IAM/identity-access-management.html {
+             return 301 /docs/linux/administration/identity-access-management.html;
```

Comment:
Why redirecting to linux and not kubernetes?
Are we not kubernetes first?

Reply:
baremetal is Linux-specific, so those pages should all continue pointing at Linux platform as it's most relevant to that page.

*But* min.io/docs will default to Kubernetes.